nvflare/dashboard/cli.py

```diff
@@ -157,18 +168,18 @@ def define_dashboard_parser(parser):
         "--passphrase", help="Passphrase to encrypt/decrypt root CA private key.  !!! Do not share it with others. !!!"
     )
     parser.add_argument("-e", "--env", action="append", help="additonal environment variables: var1=value1")
+    parser.add_argument("--cred", help="set credential directly in the form of USER_EMAIL:PASSWORD")
 
 
 def handle_dashboard(args):
-    if has_no_arguments():
-        print("Add -h option to see usage")
-        exit(0)
     if args.stop:
         stop()
     elif args.start:
         start(args)
-    elif args.cloud == "azure":
+    elif args.cloud in supported_csp:
         cloud(args)
```

Comment:
I notice we already check "args.cloud in supported_csp" here,
in that case should we move the warning message here?

Reply:
Good suggestion.  Let me move that warning here.